content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -389,7 +389,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
    for more details.
 - `BoundServiceAccountTokenVolume`: Migrate ServiceAccount volumes to use a projected volume consisting of a
    ServiceAccountTokenVolumeProjection.
-   Check [Service Account Token Volumes](https://git.k8s.io/community/contributors/design-proposals/storage/svcacct-token-volume-source.md)
+   Check [Bound Service Account Tokens](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md)
```

Comment:
For beta, I would expect a section added to https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account detailing the beta feature, the metrics to monitor to ensure workloads are happy with the rotated tokens, and use of the `--service-account-extend-token-expiration=false` flag once all workloads are verified happy.

This PR should serve as the placeholder docs PR for the BoundServiceAccountTokenVolume Beta feature